### DISCUSSION TITLE
Error when following the get started guide

### DISCUSSION THREAD

**Discussion by rassidm** [fluxcd/flux2#886] [Q&A]
after running  flux bootstrap github \
  --owner=$GITHUB_USER \
  --repository=fleet-infra \
  --branch=main \
  --path=./clusters/my-cluster \
  --personal
  
I get this error :

verifying installation
notification-controller: unhealthy (timed out waiting for rollout)
source-controller: unhealthy (timed out waiting for rollout)
kustomize-controller: unhealthy (timed out waiting for rollout)
install failed

I'm using Kubernetes kind
https://kind.sigs.k8s.io/

flux check --pre 
works fine 

Can uou please help 

Thank you

**Comment by brianpham** [comment]
Did you check to make sure you are using the correct context and that your kind cluster came up properly? You could always specify the context with `--context <cluster_context>` as part of the bootstrap command.

**Reply by rassidm** [reply]
Yes I checked, kind is configured as expected 

I will go with **Argo CD** its a more stable project a features are well tested before getting released.

**Reply by hiddeco** [reply]
> I will go with Argo CD its a more stable project a features are well tested before getting released.

Our end-to-end tests literally run on `kind`, [including the end-to-end tests that connect to GitHub](https://github.com/fluxcd/flux2/blob/main/.github/workflows/bootstrap.yaml).

**Reply by hiddeco** [reply]
To answer your question: if you are running on a machine that is slower, you may need to increase the `--timeout` (default `5m0s`) to a higher value.

It would also help if you would e.g. provide some information about the state the controllers are in after the `bootstrap` command returns an error, as we have very little to go with here.

**Reply by aalstes** [reply]
Adding a --timeout=20m worked for me. Thanks!

**Reply by aalstes** [reply]
Tip: checking pod statuses in the flux-system namespace showed me they were not ready yet after the 5m timeout.
